### ISSUE TITLE
[Feature] Please add keyboard shortcuts to scroll up and down

### ISSUE THREAD

**Issue by bwhurd** [Bin-Huang/chatbox#916]
**Problem Description**
I'd like keyboard shortcuts to scroll up and down.

**Proposed Solution**
Alt+A:
On first press: Scroll to the beginning of the current message, aligning its top with the window's top.
On subsequent presses: Scroll to the start of the preceding message, one-at-a-time.

Alt+D:
Scroll down so the top of the next message aligns with the window's top.

Alt+E:
Scroll to end of current conversation.

Alt+T:
Scroll to top of current conversation.

**Comment by jordanbates** [comment]
Agreed that shortcuts to jump between responses would be very useful. I find myself doing lots of scrolling to get to past responses or just to get to the top of the response to copy a long code block sometimes.
